src/styles/blocks/about.scss

```diff
@@ -0,0 +1,45 @@
+.about {
+  margin: 120px 0 110px;
+
+  @include onTablet {
+    margin-bottom: 96px;
+  }
+
+  @include onDesktop {
+    margin-bottom: 120px;
+  }
+
+  &__title {
+    @include title;
+    margin: 0 0 32px;
+    text-align: left;
+
+    @include onTablet {
+      font-size: 40px;
+      line-height: 40px;
+    }
+
+    @include onDesktop {
+      font-size: 44px;
+      line-height: 48px;
+    }
+  }
+
+  &__bullet-points {
+    margin-bottom: 40px;
+
+    &--text {
+      margin: 0;
+      font-weight: 700;
+      line-height: 22px;
+
+      &:not(:last-child) {
+        margin-bottom: 16px;
+      }
+    }
+  }
+
+  &__description {
+    line-height: 27px;
```

Comment:
Similar to the previous suggestions, consider using a variable for the line-height. You can create a variable like $line-height-medium: 27px; in your variables.scss file and use it here. 👍